### ISSUE TITLE
New team member onboarding checklist for @nateborr

### ISSUE THREAD

**Issue by edwintorres** [18F/tock#1801]
## Team member checklist
Before you get started onboarding, your Tock developer partner needs to take care of a few things.
They will [create a new issue](https://github.com/18f/tock/issues/new) with the following
checklist, work through it with you.
- [ ] Generate a Tock API authorization token in the admin for the new developer.
- [x] [Edwin initiated] Grant access to New Relic.
- [x] Grant access to the GitHub repository.
- [x] Grant access to SpaceDeveloper roles on Cloud.gov for `staging` and `prod` spaces within the  `gsa-18f-tock` org.
- [x] Send invitation to #tock-dev in Slack.
- [ ] Grant access to `GSA_TE_TockLibrary` and `GSA_TE_UtilitiesLibrary` scripts.
- [ ] Grant access to `gsa-18f-tock` org on Snyk.

## New developer checklist
You will need to be sure you have:
- [ ] Familiarized yourself with the systems and scripts your onboarding partner has granted access to
- [x] Reviewed the [Tock deployment process](https://github.com/18f/tock/tree/main/docs/deployment-process.md).

**Comment by nateborr** [comment]
@edwintorres I've reviewed what I can before I get access to the Google Scripts and Snyk.

**Comment by nateborr** [comment]
Waiting on Edwin's return for this. I'd like access to the Tock API in order to validate the updates for #1796.

**Comment by amandacostello** [comment]
@nateborr Can we close this issue since you have #1822 as a follow-up to #1796 ?

**Comment by nateborr** [comment]
I have enough to work with for the present round of Tock related issues and I'll be offboarding from TLC crew next week, so I'm closing this out as done enough!
